src/utils/objectUtils.ts

```diff
@@ -196,24 +196,29 @@ export function convertDateFieldsToIsoString<Input extends object>(
     return object.map(convertDateFieldsToIsoStringAux)
   }
 
-  return Object.entries(object).reduce((result, [key, value]) => {
-    // @ts-ignore
-    result[key] = convertDateFieldsToIsoStringAux(value)
-    return result
-  }, {} as ExactlyLikeWithDateAsString<Input>)
+  return Object.entries(object).reduce(
```

Comment:
original implementation violated "no destructuring of the accumulator" perf rule